src/components/common/ColonyHome/ColonyNavigation/NavItem.tsx

```diff
@@ -18,44 +19,54 @@ interface Props {
 
 const displayName = 'dashboard.ColonyHome.ColonyNavigation.NavItem';
 
+const handleLinkKeyDown = (
+  evt: KeyboardEvent<HTMLAnchorElement>,
+  disabled: boolean,
+) => {
+  if (disabled && evt.key === ENTER) {
+    evt.preventDefault();
+  }
+};
+
+const getClassNames = (
+  showDot: boolean,
+  linkTo: string,
+  currentLocation: string,
+) => {
+  const classNames = [styles.main];
+
+  if (showDot) {
+    classNames.push(styles.showDot);
+  }
+
+  if (linkTo === currentLocation) {
```

Comment:
Ensures link is styled as active when at the correct route but if the button itself isn't active, e.g. after page refresh